packages/core/lib/console.js

```diff
@@ -284,9 +282,10 @@ class Console extends EventEmitter {
   }
 
   interpret(input, context, filename, callback) {
+    // processInput returns a sanitized string
     const processedInput = processInput(input);
     if (
-      this.command.getCommand(processedInput, this.options.noAliases) != null
+      getCommand(processedInput.split(" "), {}, this.options.noAliases) !== null
```

Comment:
Are quoted strings a concern?

Reply:
I'm not sure, can you give an example of what you mean?

Reply:
Command line arguments aren't strictly whitespace separated and should account for quoted arguments.  Here's a pretty contrived example.

```
"--breakfast 'ham and eggs' "
split by space:  [ '--breakfast', "'ham", 'and', "eggs'", '' ]
```

Reply:
I think this problem must exist in current Truffle as well.

Reply:
The changes introduce a split on space though.

Reply:
It only changes where the split is performed. [Here](https://github.com/trufflesuite/truffle/blob/develop/packages/core/lib/command.js#L117-L119) is where it is done in the current code. This will change what `getCommand` receives but shouldn't affect the outcome since that function is not concerned with anything but the first string.

Reply:
Oh, it used a split on space-sep before. :+1: